Upgrade 0.16.1 (#1)
GroestlCoin/groestlcoinj: README.md

> [![Github Build Status](https://github.com/bitcoinj/bitcoinj/workflows/Java%20CI/badge.svg)](https://github.com/bitcoinj/bitcoinj/actions)

Reviewer: ```suggestion
[![Github Build Status](https://github.com/Groestlcoin/groestlcoinj/workflows/Java%20CI/badge.svg)](https://github.com/Groestlcoin/groestlcoinj/actions)
```

> [![GitLab Build Status](https://gitlab.com/bitcoinj/bitcoinj/badges/master/pipeline.svg)](https://gitlab.com/bitcoinj/bitcoinj/pipelines)
> [![Coverage Status](https://coveralls.io/repos/bitcoinj/bitcoinj/badge.png?branch=master)](https://coveralls.io/r/bitcoinj/bitcoinj?branch=master)

Reviewer: ```suggestion
[![GitLab Build Status](https://gitlab.com/Groestlcoin/groestlcoinj/badges/master/pipeline.svg)](https://gitlab.com/Groestlcoin/groestlcoinj/pipelines)
[![Coverage Status](https://coveralls.io/repos/Groestlcoin/groestlcoinj/badge.png?branch=master)](https://coveralls.io/r/Groestlcoin/groestlcoinj?branch=master)
```

Author: We are not using GitLab for anything, we could remove it  -- or we could also publish to GitLab.  Also, we don't use coveralls.io, but we could add support for it.

Reviewer: In that case lets remove Gitlab and coveralls

> [![Visit our IRC channel](https://kiwiirc.com/buttons/irc.freenode.net/bitcoinj.png)](https://kiwiirc.com/client/irc.freenode.net/bitcoinj)

Reviewer: ```suggestion
[![Visit our IRC channel](https://kiwiirc.com/nextclient/irc.libera.chat/groestlcoin.png)](https://kiwiirc.com/nextclient/irc.libera.chat/groestlcoin)
```

Alternatively, just import the project using your IDE. [IntelliJ](http://www.jetbrains.com/idea/download/) has Gradle integration built-in and has a free Community Edition. Simply use `File | New | Project from Existing Sources` and locate the `build.gradle` in the root of the cloned project source tree.

### Building and Using the Wallet Tool

> The **bitcoinj** `tools` subproject includes a command-line Wallet Tool (`wallet-tool`) that can be used to create and manage **bitcoinj**-based wallets (both the HD keychain and SPV blockchain state.) Using `wallet-tool` on Bitcoin's test net is a great way to learn about Bitcoin and **bitcoinj**.
> 

Reviewer: ```suggestion
The **groestlcoinj** `tools` subproject includes a command-line Wallet Tool (`wallet-tool`) that can be used to create and manage **groestlcoinj**-based wallets (both the HD keychain and SPV blockchain state.) Using `wallet-tool` on Groestlcoin's test net is a great way to learn about Groestlcoin and **groestlcoinj**.

```
